protocol.md

```diff
@@ -1,10 +1,11 @@
 # tus resumable upload protocol
 
-**Version:** 0.2.2 ([SemVer](http://semver.org))<br>
+**Version:** 1.0.0 ([SemVer](http://semver.org))<br>
 **Date:** 2014-01-26<br>
```

Comment:
Thanks, I had a note by myself. I will be updated before the PR is merged into master.